autogen.sh

```diff
@@ -1,2 +1,8 @@
 #!/bin/sh
-autoreconf -i
+
+set -e
+
+# (Re)Generate autotools files
+mkdir -p autoconf
```

Comment:
It is the use of the `AC_CONFIG_AUX_DIR()` macro in `configure.ac` that determines that auxiliary files should be put in the directory `autoconf` (thus this path should not be hardcoded here). Also, are there any versions of `autoreconf` that do not create this directory if needed?

Reply:
I have seen it in many `autogen.sh` scripts but I haven't checked whether all the `autoreconf` versions create this directory or not. However, I have tried in Ubuntu and OpenSUSE and it gets automatically generated.

Thus, I have removed the directory creation and the `autoreconf` directory from the repository (since it is a build folder that will be automatically created). The files inside it where already ignored.